bot-twitter/src/index.ts

```diff
@@ -15,7 +16,11 @@ const loop = async function (): Promise<void> {
         clearInterval(refetchIntervalId);
     }
     try {
-        (await getNewAuctions(ETHEREUM_NETWORK)).map(notify);
+        const activeAuctions = await getAllActiveAuctions(ETHEREUM_NETWORK);
+        const newAuctions = getNewAuctionsFromActiveAuctions(activeAuctions);
+
+        activeAuctions.map(auction => participate(auction));
```

Comment:
Where/how do we handle errors happening during `participate` call?

Reply:
Currently they just get printed in the console. If they are actual thrown errors, the participation is canceled.

Reply:
There is a different between handled and unhandled exceptions and usually there should be no unhandled ones:
```suggestion
        activeAuctions.map(auction => participate(auction).catch(error => console.error(`participation error: ${error.message}`)));
```

The top try catch here will not catch those errors cause they are not awaited for (and we don't want to await them since two bots should work independently)